proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -53,3 +53,13 @@ message Experiment {
     // The username of the user that created the experiment.
     string username = 10;
 }
+
+// ValidationHistoryEntry is a single entry for a validation history for an experiment.
+message ValidationHistoryEntry {
+    // The id for the trial associated with this validation entry.
+    int32 trial_id = 1;
```

Comment:
could you please add the required annotation option for fields that would always be present on the response? which would be all 3 in this case.

we should do this for all endpoints we add/modify. (I added it to sync agenda)